### PR TITLE
JSCS test all modules

### PR DESCRIPTION
**Pull request by WebSeed**

### CODE REVIEW

**Comment by WebSeed** [comment]
@MrSwitch another fairly big reformatting exercise. Thought you should take a look before the merge (or merge yourself if you like). I've also committed in the `dist` - do you want these excluded?

**Comment by MrSwitch** [comment]
Great,  could you remove the dist/ changes though. There's a bug in IE8 and best if people don't get that version from dist directory. I've fixed the issue in branch hello-polyfill

**Comment by WebSeed** [comment]
> Great, could you remove the dist/ changes though

Done.

File: src/modules/dropbox.js [hunk 1]
```diff
@@ -1,213 +1,216 @@
...
+					o.name = o.display_name;
+					o.first_name = o.name.split(' ')[0];
+					o.last_name = o.name.split(' ')[1];
+					o.id = o.uid;
+					delete o.uid;
+					delete o.display_name;
+					return o;
+				},
+
+				default: function(o, headers, req) {
```
Comment (MrSwitch) [inline src/modules/dropbox.js]: `default` is a [reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords) and needs to be in quotes here.

Comment (WebSeed) [inline src/modules/dropbox.js]: Yeah, I was uneasy about this - with the current config JSCS complains about this with quotes and the unquoted version is valid according to the spec (as these are identifier names, not identifiers). Does this cause problems in some browsers in your experience - older IE perhaps? Regardless, I'll disable this option in JSCS in I can and revert.

Comment (MrSwitch) [inline src/modules/dropbox.js]: Yup, IE8 throws a syntax error.

Its easy to replace "default" with "*" etc, but requires a core change, so best left to another commit.

File: .jscsrc [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
   "validateIndentation": "\t",
   "disallowMultipleSpaces": true,
+  "disallowQuotedKeysInObjects": "allButReserved",
```
Comment (WebSeed) [inline .jscsrc]: This is what I was looking for before - allows certain properties to be quoted such as 'default'.

**Comment by WebSeed** [comment]
OK @MrSwitch - quoted 'default's

File: .jscsrc (hunk 2 above)
Comment (MrSwitch) [inline .jscsrc]: Perfect!

**Pull request merged**